Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.py

```diff
@@ -707,12 +728,13 @@ def _get_email_attachments(self, message_id, user_id=None, overwrite_rate_limit_
             attachment_type = attachment.get('@odata.type', '')
             attachment_name = attachment.get('name', 'untitled_attachment')
 
-            try:
-                demisto.debug(f"Trying to decode the attachment file name: {attachment_name}")
-                attachment_name = base64.b64decode(attachment_name)
-            except Exception as e:
-                demisto.debug(f"Could not decode the {attachment_name=}: error: {e}")
-                pass
+            if not self._is_only_ascii(attachment_name):
+                try:
+                    demisto.debug(f"Trying to decode the attachment file name: {attachment_name}")
+                    attachment_name = base64.b64decode(attachment_name)
+                except Exception as e:
+                    demisto.debug(f"Could not decode the {attachment_name=}: error: {e}")
```

Comment:
shouldn't this be an error, rather than `debug`?

Reply:
We agreed to leave it at that.